install.sh

```
@@ -1796,171 +1820,414 @@ install_font() {
+install_desktop_scripts() {
+    local de_name="XFCE"
+    local msg_config_key="MSG_CONFIG_XFCE_SCRIPTS"
+
+    # Read desktop config
+    if [ -f "$OHMYTERMUX_CONFIG_DIR/desktop.conf" ]; then
+        source "$OHMYTERMUX_CONFIG_DIR/desktop.conf"
+    fi
+
+    case "${DESKTOP_SESSION:-xfce}" in
+        lxqt)
+            de_name="LXQt"
```

Shellcheck (0.11.0)

[warning] de_name appears unused. Verify use (or export if used externally).

(SC2034)